src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/diagnostics/rules/MissingAnnotationsDiagnosticRule.java

```diff
@@ -7,61 +7,60 @@
 
 import java.util.concurrent.atomic.AtomicInteger;
 
-import static com.azure.tools.apiview.processor.analysers.util.ASTUtils.getClasses;
-import static com.azure.tools.apiview.processor.analysers.util.ASTUtils.getPublicOrProtectedMethods;
-import static com.azure.tools.apiview.processor.analysers.util.ASTUtils.makeId;
-
+import static com.azure.tools.apiview.processor.analysers.util.ASTUtils.*;
 import static com.azure.tools.apiview.processor.model.DiagnosticKind.*;
 
 public class MissingAnnotationsDiagnosticRule implements DiagnosticRule {
 
     @Override
     public void scanIndividual(final CompilationUnit cu, final APIListing listing) {
-        getClasses(cu).forEach(typeDeclaration -> {
-            String className = typeDeclaration.getNameAsString();
-
-            if (className.endsWith("Builder")) {
-                // check if @ServiceClientBuilder annotation is present
-                if (!typeDeclaration.isAnnotationPresent("ServiceClientBuilder")) {
-                    listing.addDiagnostic(new Diagnostic(
-                        INFO,
-                        makeId(cu),
-                        "Classes named *Builder are potential candidates to have the @ServiceClientBuilder annotation applied.",
-                        "https://azure.github.io/azure-sdk/java_introduction.html#service-client-creation"));
-                }
-            } else if (className.endsWith("Client")) {
-                // check if the @ServiceClient annotation is present
-                if (!typeDeclaration.isAnnotationPresent("ServiceClient")) {
-                    listing.addDiagnostic(new Diagnostic(
-                        INFO,
-                        makeId(cu),
-                        "Classes named *Client are potential candidates to have the @ServiceClient annotation applied.",
-                        "https://azure.github.io/azure-sdk/java_introduction.html#service-client"));
-                }
+        getClasses(cu)
+            .filter(type -> getPackageName(type).equals("com.azure"))   // we only want to give this guidance to Azure SDK developers
```

Comment:
Should this be `.startsWith("com.azure")` instead of `equals`?

Reply:
Good call - I'll make that change